config/application.yml.default

```diff
@@ -116,6 +116,8 @@ idv_max_attempts: 5
 idv_min_age_years: 13
 idv_native_camera_a_b_testing_enabled: false
 idv_native_camera_a_b_testing_percent: 10
+idv_acuant_sdk_upgrade_a_b_testing_enabled: true
```

Comment:
Do we really want this enabled by default? I would expect a test to be disabled by default and explicitly enabled per environment.

Reply:
I'll switch back and you can walk me through the typical process